### PR TITLE
Fix invalid Barrier transactions

### PR DESCRIPTION
**Pull request by sagar-solana**
#### Problem

Barrier Txs' "from_keypair" and "to_pubkey " were set to the same account. This is explicitly disallowed by the runtime. 

#### Summary of Changes

Send barrier txs to a different account.

### CODE REVIEW

File: bench-tps/src/main.rs [hunk 1]
```diff
@@ -146,7 +146,9 @@ fn main() {
         target /= MAX_SPENDS_PER_TX;
     }
     let gen_keypairs = rnd.gen_n_keypairs(total_keys as u64);
-    let barrier_id = rnd.gen_n_keypairs(1).pop().unwrap();
+    let mut barrier_ids = rnd.gen_n_keypairs(2);
```
Comment (mvines) [inline bench-tps/src/main.rs]: While we’re here, an id is a Pubkey not a Keypair.  You could rename barrier_id to barrier_keypair, and barrier_dest_id to barrier_dest_keypair to resolve

File: bench-tps/src/main.rs [hunk 2]
```diff
@@ -146,7 +146,9 @@ fn main() {
...
+    let barrier_source_id = barrier_ids.pop().unwrap();
```
Comment (CriesofCarrots) [inline bench-tps/src/main.rs]: Seems like you could do this in 2 lines instead of 3 by using Keypair::new()...?

File: bench-tps/src/main.rs (hunk 1 above)
Comment (sagar-solana) [reply bench-tps/src/main.rs]: Barrier_dest_id is a pubkey. I'll update the barrier_id to barrier_keypair though

Comment (mvines) [reply bench-tps/src/main.rs]: barrier_source_id too please

**Pull request merged**